docs/en/ingest-management/elastic-agent/running-on-kubernetes-gke-standard-managed-by-fleet.asciidoc

```diff
@@ -0,0 +1,170 @@
+[[running-on-kubernetes-gke-standard-managed-by-fleet]]
+= Run {agent} on GKE standard managed by {fleet}
+
+Use {agent} https://www.docker.elastic.co/r/beats/elastic-agent[Docker images] on Kubernetes to
+retrieve cluster metrics.
+
+TIP: Running {ecloud} on Kubernetes? Refer to {eck-ref}/k8s-elastic-agent-fleet.html[Run {elastic-agent} on ECK].
+
+ifeval::["{release-state}"=="unreleased"]
+
+However, version {version} of {agent} has not yet been
+released, so no Docker image is currently available for this version.
+
+endif::[]
+
+[discrete]
+== Important Notes:
+
+On managed Kubernetes solutions like GKE, {agent} has no access several data sources. Find below the list of
+the non available data:
+
+1. Metrics from https://kubernetes.io/docs/concepts/overview/components/#control-plane-components[Kubernetes control plane]
+components are not available. Consequently metrics are not available for `kube-scheduler` and `kube-controller-manager`.
+In this regard, the respective dashboards will not be populated with data.
+2. Audit logs are available only on Kubernetes master nodes as well, hence cannot be collected by {agent}.
+
+Also note that the documentation page only applies to Standard GKE and not Autopilot. Autopilot support is a work in progress
+at the time of writing these docs.
+
+[discrete]
+== Prerequisites
+
+Kube-state-metrics is not deployed by default in Kubernetes GKE!
+
+Data_streams with the `state_` prefix require `kube-state-metrics` to be present.
+
+In order to install kube-state-metrics follow the instructions for its deployment are available
+https://github.com/kubernetes/kube-state-metrics#kubernetes-deployment[here].
+Generally `kube-state-metrics` runs a `Deployment` and is accessible via a service called `kube-state-metrics` on
+`kube-system` namespace, which will be the service to use in our configuration of agent
+service within the cluster.
+
+[discrete]
+== Kubernetes deploy manifests
+
+With {fleet}, each agent enrolls in a policy defined in {kib} and stored in
+{es}. The policy specifies how to collect observability data from the services
+to be monitored. The {agent} connects to a trusted {fleet-server} instance
+to retrieve the policy and report agent events.
+
+We recommend using {fleet} management because it makes the management and
+upgrade of agents' integrations considerably easier.
```

Comment:
>With {fleet}, each agent enrolls in a policy defined in {kib} and stored in
{es}. The policy specifies how to collect observability data from the services
to be monitored. The {agent} connects to a trusted {fleet-server} instance
to retrieve the policy and report agent events.
We recommend using {fleet} management because it makes the management and
upgrade of agents' integrations considerably easier.

I think all this part should go first to the section `Configure a Fleet Policy`

Reply:
Also the rest section of `Kubernetes deploy manifests` should go and be merged with the `Deploy` section

What do you think?

Reply:
Also last comment on this:
Installation Steps can be the [discrete] header here and under that to have:
1. Configure a Fleet policy
2. Enroll Elastic Agent to Fleet
3. Deploy kubernetes manifests

Those I dont think should be discrete and I would put them as sub headers with the numbers as well. To so that are steps in a sequence